shift_beneficiary/models/shift_shift.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class ShiftShift(models.Model):
+    _inherit = "shift.shift"
+
+    template_beneficiary = fields.Many2one(
+        "res.partner",
+        related="task_template_id.beneficiary",
+        store=True,
+    )
```

Comment:
I see, do you think I should rewrite the code so that beneficiary_id behaves like task_type_id ? Or can we go with this design and refactor afterward if needed ?

Reply:
This should be discussed with Pol. Because it's a matter of edge case and usage of the module. I can be part of the discussion to explain.

But the refacto can be done later, it will just need a migration script. You may write a note in the README about this limitation for now.